app/modules/annotations/models.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""
+Annotations database models
+--------------------
+"""
+
+from sqlalchemy_utils import Timestamp
+
+from app.extensions import db
+
+import uuid
+
+
+class Annotation(db.Model, Timestamp):
+    """
+    Annotations database model.
+    """
+
+    guid = db.Column(
+        db.GUID, default=uuid.uuid4, primary_key=True
+    )  # pylint: disable=invalid-name
+
+    asset_guid = db.Column(
+        db.GUID,
+        db.ForeignKey('asset.guid', ondelete='CASCADE'),
```

Comment:
while trying to fully grasp the cascade behavior here, i ran across [this in these docs](https://docs.sqlalchemy.org/en/14/orm/cascades.html#using-foreign-key-on-delete-cascade-with-orm-relationships):

> In order to use ON DELETE foreign key cascades in conjunction with relationship(), it’s important to note first and foremost that the relationship.cascade setting must still be configured to match the desired “delete” or “set null” behavior

not saying i 100% grasp this (or even 90%) but i think it might mean the `relationship()` below might need some more `cascade=()` stuff?

anyway, as long as the _results_ are such that when an Asset is deleted, its Annotations go with it, i am satisfied.

Reply:
I took the submissions/assets relationship as a template. 
I'll tweak the create REST API so that it must contain an asset GUID, then will try to delete the annotation and see what blows up